src/main/kotlin/de/danielmaile/mpp/item/recipe/recipes/ArmorRecipe.kt

```diff
@@ -0,0 +1,87 @@
+/*
+ * This file is part of MPP.
+ * Copyright (c) 2022-2023 by it's authors. All rights reserved.
+ * MPP is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * MPP is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with MPP.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package de.danielmaile.mpp.item.recipe.recipes
+
+import de.danielmaile.mpp.item.recipe.CraftingRecipe
+import de.danielmaile.mpp.item.recipe.ShapedRecipe
+import org.bukkit.inventory.ItemStack
+import org.bukkit.inventory.Recipe
+
+abstract class ArmorRecipe : CraftingRecipe() {
```

Comment:
Why is ArmorRecipe and ToolRecipe in the recipes package but for ex. FurnaceRecipe not?

Reply:
I added armor and tool recipes as a recipe sub class as it gives "predefined" recipes, didn't add furnaces specifically because it's more like saying ShapedRecipe but instead FurnaceRecipe. Does that make sense?

Armor and tools only require 1 (2) values, a single material and occasionally stick (+ result), and furnace requires material, time, xp, result... Do you agree?